### ISSUE TITLE
Helix is slow in SSHFS

### ISSUE THREAD

**Issue by SeresHotes** [helix-editor/helix#11031]
Hello! I can't use Helix in the remote machine, because it has a very glibc, and helix just doesn't want to run. That's why I mounted a directory from the remote machine using SSHFS on my local Arch/Hypr machine. Ping is about 100-200ms.

The setup works perfectly fine using terminal navigation and even LF - terminal file manager. But when I open Helix in some directory and try to <space>-f to open file navigation, it loads the files very slowly. The same problem occures when I use :o command to open a file - each character is written very slowly (i.e. if I write ":o config.txt" in a second, Helix prints one letter only every 10 seconds after :o).

**Comment by TornaxO7** [comment]
May I ask if you've disabled your `LSP`? I could imagine that LSPs could be a reason why helix's response is so slow due to the LSP.

**Comment by SeresHotes** [comment]
Yes, LSP is disabled. I don't even need to have a file opened. I enter "hx" in terminal, an empty buffer opens and file picker as well as writting ":o" command. Everything else is fine - I can open multiple files, edit them, save them without and slowness. So, it feels like a problem on getting list of files or something for the file system.
